From dc48abafa1438ab75c41973b4d670038a3126ece Mon Sep 17 00:00:00 2001
From: Li Wang <liwang@redhat.com>
Date: Mon, 11 Jul 2016 19:57:25 +0800
Subject: [PATCH] zram01: extend the zram3 disksize to 40MB

It failed to format btrfs on /dev/zram3 device since the
disk size is too small (btrfs-progs is v4.4.1).

The error msg we catched:

zram01 3 TINFO : make ext3 filesystem on /dev/zram0
zram01 3 TINFO : make ext4 filesystem on /dev/zram1
zram01 3 TINFO : make xfs filesystem on /dev/zram2
zram01 3 TINFO : make btrfs filesystem on /dev/zram3
'/dev/zram3' is too small to make a usable filesystem
Minimum size for each btrfs device is 41943040.
btrfs-progs v4.4.1
See http://btrfs.wiki.kernel.org for more information.
zram01 3 TFAIL : failed to make btrfs on /dev/zram3
zram01 4 TINFO : zram cleanup

Signed-off-by: Li Wang <liwang@redhat.com>
Reviewed-by: Cyril Hrubis <chrubis@suse.cz>
Reviewed-by: Alexey Kodanev <alexey.kodanev@oracle.com>
---
 testcases/kernel/device-drivers/zram/zram01.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/testcases/kernel/device-drivers/zram/zram01.sh b/testcases/kernel/device-drivers/zram/zram01.sh
index 26dae90..caa0e05 100755
--- a/testcases/kernel/device-drivers/zram/zram01.sh
+++ b/testcases/kernel/device-drivers/zram/zram01.sh
@@ -39,8 +39,8 @@ zram_max_streams="2 3 5 8"
 # not support mem suffixes, in some newer kernels, they use
 # memparse() which supports mem suffixes. So here we just use
 # bytes to make sure everything works correctly.
-zram_sizes="26214400 26214400 26214400 26214400" # 25MB
-zram_mem_limits="25M 25M 25M 25M"
+zram_sizes="26214400 26214400 26214400 41943040" # 25MB, 40MB for btrfs
+zram_mem_limits="25M 25M 25M 40M"
 zram_filesystems="ext3 ext4 xfs btrfs"
 zram_algs="lzo lzo lzo lzo"
 
-- 
1.8.3.1

